README.md

```diff
@@ -1,2 +1,86 @@
 # regional-planning
-ArcGIS and WMS map layer plugin for the GeositeFramework
+
+The `regional-planning` plugin is an updated version of the [map-tree](https://github.com/CoastalResilienceNetwork/map-tree/) plugin (formerly known as `layer-selector`). It has many of the same features, but with an updated UI, simplified configuration, and an integrated version of the [draw-and-report](https://github.com/CoastalResilienceNetwork/draw-and-report) plugin.
+
+### Getting started
+
+The easiest way to see how to configure the plugin is to take a look at [sample_layers.json](https://github.com/CoastalResilienceNetwork/regional-planning/blob/master/sample_layers.json). This configuration file exercises all of the of plugin's functionality. It initially started as updated version of the map-tree configuration for the [New Jersey region](https://github.com/CoastalResilienceNetwork/newjersey-region/blob/master/plugins/layer_selector/layers.json), but grew to test out more functionality.
+
+The biggest difference between the configuration of the `map-tree` plugin and the `regional-planning` plugin is that **layers are identified by name and not id**. If you update the name of the layer in the map services, you must also update the configuration for any region using that layer. Also, layers with duplicate names will cause problems with this plugin and should be avoided.
```

Comment:
This is a good thing to call out, we should add that it was done to allow a reordering of layers within the service without having to update the config of the layers.